template/{{cookiecutter.module_name}}/1.0/{{cookiecutter.module_name}}.smk

```diff
@@ -98,7 +120,7 @@ rule _{{cookiecutter.module_name}}_output_{{cookiecutter.output_file_type}}:
     output:
         {{cookiecutter.output_file_type}} = CFG["dirs"]["outputs"] + "{{cookiecutter.output_file_type}}/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.output.filt.{{cookiecutter.output_file_type}}"
     run:
-        op.relative_symlink(input.{{cookiecutter.output_file_type}}, output.{{cookiecutter.output_file_type}})
+        op.relative_symlink(input.{{cookiecutter.output_file_type}}, output.{{cookiecutter.output_file_type}}, in_module= True)
```

Comment:
What is the rationale of using the `in_module` feature only for the "per tumour" mode, and not for the modules which will run per sample?

Reply:
Error on my side, accidently missed it